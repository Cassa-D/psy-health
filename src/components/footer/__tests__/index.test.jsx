import React from 'react';
import Footer from '../../footer';
import renderer from 'react-test-renderer';

it('footer-renderer', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
});