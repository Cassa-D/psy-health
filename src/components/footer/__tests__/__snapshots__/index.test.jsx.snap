// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer-renderer 1`] = `
<footer
  className="sc-bdnylx tFLML"
>
  <div
    className="ant-row sc-gtssRu gCLzAC"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24"
      style={Object {}}
    >
      <a
        className="sc-eCApGN lfgKha"
        href="#"
      >
        <img
          alt="Logo"
          src="logo-azul.svg"
        />
      </a>
    </div>
    <div
      className="ant-col sc-dlnjPT VwyOd ant-col-xs-12 ant-col-sm-6 ant-col-md-6 ant-col-lg-6 ant-col-xl-6"
      style={Object {}}
    >
      <div
        className="container-box-left"
      >
        <h4>
          Sobre a PsyHealth
        </h4>
        <a
          href="#"
        >
          Perguntas frequentes
        </a>
        <a
          href="#"
        >
          Quem somos
        </a>
        <a
          href="#"
        >
          Politica de privacidade
        </a>
        <a
          href="#"
        >
          Paciente: Termos de uso
        </a>
        <a
          href="#"
        >
          Psicólogo: Termos de uso
        </a>
      </div>
    </div>
    <div
      className="ant-col sc-dlnjPT VwyOd ant-col-xs-12 ant-col-sm-7 ant-col-md-7 ant-col-lg-7 ant-col-xl-7"
      style={Object {}}
    >
      <div
        className="container-box-middle"
      >
        <h4>
          Assuntos mais lidos
        </h4>
        <a
          href="#"
        >
          Ansiedade
        </a>
        <a
          href="#"
        >
          Depressão
        </a>
        <a
          href="#"
        >
          Sindrome do impostor
        </a>
        <a
          href="#"
        >
          Bipolaridade
        </a>
        <a
          href="#"
        >
          Estresse
        </a>
      </div>
    </div>
    <div
      className="ant-col sc-dlnjPT VwyOd ant-col-xs-12 ant-col-sm-7 ant-col-md-7 ant-col-lg-7 ant-col-xl-7"
      style={Object {}}
    >
      <div
        className="container-box-middle"
      >
        <h4>
          Especialistas
        </h4>
        <a
          href="#"
        >
          Psicanalistas
        </a>
        <a
          href="#"
        >
          Psicólogos
        </a>
        <a
          href="#"
        >
          Terapeutas
        </a>
        <a
          href="#"
        >
          Coachs
        </a>
      </div>
    </div>
    <div
      className="ant-col sc-hKFyIo jnTjZl ant-col-xs-12 ant-col-sm-4 ant-col-md-4 ant-col-lg-4 ant-col-xl-4"
      style={Object {}}
    >
      <div
        className="container-icons-social-network"
      >
        <h4>
          Redes sociais
        </h4>
        <div>
          <a
            href="#"
          >
            <span>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224,202.66A53.34,53.34,0,1,0,277.36,256,53.38,53.38,0,0,0,224,202.66Zm124.71-41a54,54,0,0,0-30.41-30.41c-21-8.29-71-6.43-94.3-6.43s-73.25-1.93-94.31,6.43a54,54,0,0,0-30.41,30.41c-8.28,21-6.43,71.05-6.43,94.33S91,329.26,99.32,350.33a54,54,0,0,0,30.41,30.41c21,8.29,71,6.43,94.31,6.43s73.24,1.93,94.3-6.43a54,54,0,0,0,30.41-30.41c8.35-21,6.43-71.05,6.43-94.33S357.1,182.74,348.75,161.67ZM224,338a82,82,0,1,1,82-82A81.9,81.9,0,0,1,224,338Zm85.38-148.3a19.14,19.14,0,1,1,19.13-19.14A19.1,19.1,0,0,1,309.42,189.74ZM400,32H48A48,48,0,0,0,0,80V432a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V80A48,48,0,0,0,400,32ZM382.88,322c-1.29,25.63-7.14,48.34-25.85,67s-41.4,24.63-67,25.85c-26.41,1.49-105.59,1.49-132,0-25.63-1.29-48.26-7.15-67-25.85s-24.63-41.42-25.85-67c-1.49-26.42-1.49-105.61,0-132,1.29-25.63,7.07-48.34,25.85-67s41.47-24.56,67-25.78c26.41-1.49,105.59-1.49,132,0,25.63,1.29,48.33,7.15,67,25.85s24.63,41.42,25.85,67.05C384.37,216.44,384.37,295.56,382.88,322Z"
                />
              </svg>
            </span>
          </a>
          <a
            href="#"
          >
            <span>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M400 32H48A48 48 0 0 0 0 80v352a48 48 0 0 0 48 48h137.25V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.27c-30.81 0-40.42 19.12-40.42 38.73V256h68.78l-11 71.69h-57.78V480H400a48 48 0 0 0 48-48V80a48 48 0 0 0-48-48z"
                />
              </svg>
            </span>
          </a>
        </div>
        <div>
          <a
            href="#"
          >
            <span>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-48.9 158.8c.2 2.8.2 5.7.2 8.5 0 86.7-66 186.6-186.6 186.6-37.2 0-71.7-10.8-100.7-29.4 5.3.6 10.4.8 15.8.8 30.7 0 58.9-10.4 81.4-28-28.8-.6-53-19.5-61.3-45.5 10.1 1.5 19.2 1.5 29.6-1.2-30-6.1-52.5-32.5-52.5-64.4v-.8c8.7 4.9 18.9 7.9 29.6 8.3a65.447 65.447 0 0 1-29.2-54.6c0-12.2 3.2-23.4 8.9-33.1 32.3 39.8 80.8 65.8 135.2 68.6-9.3-44.5 24-80.6 64-80.6 18.9 0 35.9 7.9 47.9 20.7 14.8-2.8 29-8.3 41.6-15.8-4.9 15.2-15.2 28-28.8 36.1 13.2-1.4 26-5.1 37.8-10.2-8.9 13.1-20.1 24.7-32.9 34z"
                />
              </svg>
            </span>
          </a>
          <a
            href="#"
          >
            <span>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M186.8 202.1l95.2 54.1-95.2 54.1V202.1zM448 80v352c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V80c0-26.5 21.5-48 48-48h352c26.5 0 48 21.5 48 48zm-42 176.3s0-59.6-7.6-88.2c-4.2-15.8-16.5-28.2-32.2-32.4C337.9 128 224 128 224 128s-113.9 0-142.2 7.7c-15.7 4.2-28 16.6-32.2 32.4-7.6 28.5-7.6 88.2-7.6 88.2s0 59.6 7.6 88.2c4.2 15.8 16.5 27.7 32.2 31.9C110.1 384 224 384 224 384s113.9 0 142.2-7.7c15.7-4.2 28-16.1 32.2-31.9 7.6-28.5 7.6-88.1 7.6-88.1z"
                />
              </svg>
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="developed-by"
  >
    <span
      className="copy"
    >
      © PsyHealth
    </span>
    <div>
      <span
        className="developed-by-text"
      >
        Desenvolvido por:
      </span>
      <a
        href="https://www.linkedin.com/in/augusto-pietroski/"
      >
        Augusto,
      </a>
      <a
        href="https://www.linkedin.com/in/alexmiguel95/"
      >
        Alex,
      </a>
      <a
        href="https://www.linkedin.com/in/cassiano-doederlein-648592148/"
      >
        Cassiano,
      </a>
      <a
        href="https://www.linkedin.com/in/carolina-ahn-35ab7473/"
      >
        Carolina,
      </a>
      <a
        href="https://www.linkedin.com/in/edu-magno/"
      >
        Eduardo
      </a>
      <span>
        e
      </span>
      <a
        href="https://www.linkedin.com/in/willian-brusch-237448154/"
      >
        Willian
      </a>
    </div>
  </div>
</footer>
`;
